### PR TITLE
add proper icon and description for Gestures

### PR DESCRIPTION
**Pull request by marshmello61**
Signed-off-by: Mayur <ultramayur123@gmail.com>

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Hello! Thanks for your interest in contributing. The icons used on the main page of the Settings screen have rounded stroke terminals, but the stroke terminals of `ic_gestures.xml` appear to be sharp. I would suggest using [this icon](https://fonts.google.com/icons?selected=Material%20Icons%20Round%3Agesture%3A).

**Comment by marshmello61** [comment]
I see, thanks for reviewing. Will force push with needed change

**Comment by marshmello61** [comment]
> Hello! Thanks for your interest in contributing. The icons used on the main page of the Settings screen have rounded stroke terminals, but the stroke terminals of `ic_gestures.xml` appear to be sharp. I would suggest using [this icon](https://fonts.google.com/icons?selected=Material%20Icons%20Round%3Agesture%3A).

Done

File: lawnchair/res/drawable/ic_gestures.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:fillColor="#ffffff"
+      android:pathData="M3.72,6.04c0.47,0.46 1.21,0.48 1.71,0.06 0.37,-0.32 0.69,-0.51 0.87,-0.43 0.5,0.2 0,1.03 -0.3,1.52 -0.25,0.42 -2.86,3.89 -2.86,6.31 0,1.28 0.48,2.34 1.34,2.98 0.75,0.56 1.74,0.73 2.64,0.46 1.07,-0.31 1.95,-1.4 3.06,-2.77 1.21,-1.49 2.83,-3.44 4.08,-3.44 1.63,0 1.65,1.01 1.76,1.79 -3.78,0.64 -5.38,3.67 -5.38,5.37 0,1.7 1.44,3.09 3.21,3.09 1.63,0 4.29,-1.33 4.69,-6.1h1.21c0.69,0 1.25,-0.56 1.25,-1.25s-0.56,-1.25 -1.25,-1.25h-1.22c-0.15,-1.65 -1.09,-4.2 -4.03,-4.2 -2.25,0 -4.18,1.91 -4.94,2.84 -0.58,0.73 -2.06,2.48 -2.29,2.72 -0.25,0.3 -0.68,0.84 -1.11,0.84 -0.45,0 -0.72,-0.83 -0.36,-1.92 0.35,-1.09 1.4,-2.86 1.85,-3.52 0.78,-1.14 1.3,-1.92 1.3,-3.28C8.95,3.69 7.31,3 6.44,3c-1.09,0 -2.04,0.63 -2.7,1.22 -0.53,0.48 -0.53,1.32 -0.02,1.82zM13.88,18.55c-0.31,0 -0.74,-0.26 -0.74,-0.72 0,-0.6 0.73,-2.2 2.87,-2.76 -0.3,2.69 -1.43,3.48 -2.13,3.48z"/>
+</vector>
```
Comment (ghost) [inline lawnchair/res/drawable/ic_gestures.xml]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
    <path
        android:fillColor="#00000000"
        android:pathData="M4,6C4,4.5 6,3 7.5,4.5C10.5,7.5 2.5,12.5 5,15C7.5,17.5 11,9.5 14.5,9.5C19,9.5 18,20 14,20C10,20 11,12 20,13.5"
        android:strokeWidth="2"
        android:strokeColor="#000000"
        android:strokeLineCap="round"
        android:strokeLineJoin="round" />
</vector>
```

**Review by ghost — CHANGES_REQUESTED** [review]
It looks like the icon I linked in my last review has a non-standard stroke width, so I’ve created a similar one with the typical stroke width of 2 px. Below is a suggestion you can apply. Apologies for the oversight from my end.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Thanks, @marshmello61!

**Pull request merged**